src/__stories__/decorators.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { boolean, number, text } from '@storybook/addon-knobs/react';
+
+import { ThemeZone } from '../theme';
+
+import { DiagnosticMessagesContext, IDiagnosticMessagesProvider } from '../components/DiagnosticMessagesContext';
+
+export const Tooltips = (storyFn: Function) => {
+  const getMessages: IDiagnosticMessagesProvider = path => {
+    const summary = text('message', '');
+    if (summary === '') return [];
+    if (path.length > 0) return [];
+    return [
+      {
+        summary: text('message', ''),
+        severity: number('severity', 40),
+        severityLabel: text('severityLabel', 'warn'),
+      },
+    ];
+  };
+  return <DiagnosticMessagesContext.Provider value={getMessages}>{storyFn()}</DiagnosticMessagesContext.Provider>;
+};
+
+export const PathTooltips = (storyFn: Function) => {
+  const getMessages: IDiagnosticMessagesProvider = path => {
+    return boolean('show path tooltips', false)
+      ? [
+          {
+            summary: path.join(' > '),
+            severity: number('severity', 0),
+            severityLabel: text('severityLabel', 'info'),
```

Comment:
Could be a select of possible choices?

Reply:
I don't actually know what the possible choices are, because I'm not importing an enum from `graphite`.

Reply:
Alright, nothing compulsory to implement - just would be nice to have a set of options here so that it's easier for other developers to see what the actual options are.

Reply:
Ah.... ok I see what you mean.

Reply:
Hahaha you'll love it.
```js
severityLabel: select('severityLabel', ['warn', 'anything-other-than-warn'], 'warn'),
```

Reply:
Love it <3